src/main/java/de/fraunhofer/aisec/codyze/config/CpgConfiguration.kt

```diff
@@ -167,5 +276,21 @@ class TranslationSettings {
                 "If includes is not empty, the specified files will be excluded from being parsed and processed in the cpg. The disabled list entries always take priority over the enabled list entries. Path must be separated by \':\' (Mac/Linux) or \';\' (Windows)"],
         split = ":|;"
     )
-    var disabledIncludes: Array<File> = emptyArray()
+    fun match(value: Array<File>) {
+        matched = true
+        this.disabledIncludes = value
+    }
+
+    @Option(
+        names = ["--disabled-includes+"],
+        paramLabel = "<path>",
+        description =
+            [
+                "See --disabled-includes, but appends the values to the ones specified in configuration file."],
+        split = ":|;"
```

Comment:
Do we continue using `:|;` or should we switch to system agnostic path separators?